src/lib-components/BlockTag.vue

```diff
@@ -61,13 +64,25 @@ const parsedClasses = computed(() => {
 
 <template>
   <span :class="parsedClasses">
-    <component :is="BlockTagIcons[iconName]" v-if="iconName" class="svg" aria-hidden="true" />
-    <div class="label" v-html="label" />
+    <component
+      :is="BlockTagIcons[iconName]"
+      v-if="iconName"
+      class="svg"
+      aria-hidden="true"
+    />
+
+    <div class="label">
```

Comment:
Why was it necessary to make this change to BlockTag? This breaks the link implementation https://deploy-preview-530--ucla-library-storybook-vue3x.netlify.app/?path=/story/block-tag--html-link-ftva

We need to revert this if possible